### PR TITLE
Add yarn scripts for auto fixing lint issues.

### PR DESCRIPTION
**Pull request by AntonAM**
We already had `prettier-write` script, this PR adds fix version for the `eslint` and general `lint-fix` script that runs both prettier and eslint fixes.

### CODE REVIEW

**Review by rudream — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "test-update-snapshot": "yarn test -- --updateSnapshot",
     "tdd": "jest --watch",
     "lint": "yarn prettier-check && yarn eslint",
+    "lint-fix": "yarn prettier-write && yarn eslint-fix",
```
Comment (ryanclark) [inline package.json:21]:
```suggestion
    "lint-fix": "yarn prettier-write && yarn eslint --fix",
```

File: package.json [hunk 2]
```diff
@@ -18,7 +18,9 @@
...
     "eslint": "eslint --quiet '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
+    "eslint-fix": "eslint --fix --quiet '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
```
Comment (ryanclark) [inline package.json:23]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -18,7 +18,9 @@
...
     "type-check": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
     "prettier-check": "yarn prettier --check '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
     "prettier-write": "yarn prettier --write --loglevel silent '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
```
Comment (ryanclark) [inline package.json:26]:
```suggestion
    "prettier-write": "yarn prettier --write --log-level silent '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
```

I know you didn't change this, but it's been bugging me that `--loglevel` isn't recognised by the version of prettier we're on, so we might as well make this change here 😀

Comment (AntonAM) [reply package.json:26]: Sure 😄

**Review by ryanclark — APPROVED** [review]

**Pull request merged**